src/OrchardCore/OrchardCore.Application.Cms.Core.Targets/OrchardCore.Application.Cms.Core.Targets.targets

```diff
@@ -1,4 +1,12 @@
 <Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  
+  <PropertyGroup>
+    <_OrchardCoreBuildTasksAssembly>$(ProjectDir)..\..\OrchardCore.Build.Tasks\bin\$(Configuration)\$(CommonTargetFrameworks)\OrchardCore.Build.Tasks.dll</_OrchardCoreBuildTasksAssembly>
```

Comment:
It should be there during the build, if it's not there the GitHub Workflow will not pass

Reply:
Yeah, but this line will be executed even when one just opens the solution and anything else that MSBuild does, not just during a build.

Reply:
This will only work if the DLL already exists but if you build the solution for the first time, it won't. Won't this cause issues?>

This is weird, could you please try the following:

1. Add this reference `<ProjectReference Include="..\OrchardCore.Build.Tasks\OrchardCore.Build.Tasks.csproj" />` to `OC.Cms.Web`
2. Open CMD and locate the web project
3. Run `dotnet clean`
4. Run `dotnet build`

Reply:
Can you please make sure you test all of such scenarios? And in VS, the solution should still work with F5.